otelcol-config.yml

```diff
@@ -0,0 +1,21 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+exporters:
+  jaeger:
+    endpoint: "jaeger:14250"
```

Comment:
Is this the correct point? I think I see Jaeger listening on `16686` above, but I haven't worked much in this area...

Reply:
16686 is the jaeger UI port - this port is for the collector to export to the jaeger storage backend itself.

Reply:
Oh, also I suppose I should note that the explicit mention of 16686 up there doesn't mean that jaeger is only listening on that port - it's a docker-compose directive saying "please map port 16686 from the container network into the host network." :)